cpp/src/bitmask/bit_container.cuh

```diff
@@ -0,0 +1,239 @@
+/*
+ * Copyright (c) 2018, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef _BIT_CONTAINER_H_
+#define _BIT_CONTAINER_H_
+
+#pragma once
+
+#include <string.h>
+
+#include "cudf.h"
+#include "bit_container.h"
+#include "rmm/rmm.h"
+#include "utilities/error_utils.h"
+#include <cuda_runtime_api.h>
+
+
+/* ---------------------------------------------------------------------------- */
+/**
+ * @Synopsis  Base class for managing bit masks.
+ */
+/* ---------------------------------------------------------------------------- */
+class Util {
```

Comment:
Util is too generic a name for a class outside a namespace. This should probably be in a namespace...